app/models/RentalsAndRaRAbout.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import models.request.{BalancingCharge, ClaimExpensesOrRRR}
+import play.api.libs.json.Json
+
+final case class RentalsAndRaRAbout(
+  ukRentARoomJointlyLet: Boolean,
+  totalIncomeAmount: BigDecimal,
+  balancingCharge: BalancingCharge,
```

Comment:
balancingCharge in about section?

Reply:
![image](https://github.com/hmrc/income-tax-property/assets/56151567/577a67b7-4585-4580-9eee-e1a12800aa99)

Reply:
From the prototype, there is balancing charge as far as I can see.